cmd/thanos/downsample.go

```diff
@@ -97,25 +87,8 @@ func runDownsample(
 			cancel()
 		})
 	}
-	// Start metric and profiling endpoints.
-	{
-		mux := http.NewServeMux()
-		registerMetrics(mux, reg)
-		registerProfile(mux)
-
-		l, err := net.Listen("tcp", httpAddr)
-		if err != nil {
-			return errors.Wrapf(err, "listen on address %s", httpAddr)
-		}
-
-		g.Add(func() error {
-			return errors.Wrap(http.Serve(l, mux), "serve query")
-		}, func(error) {
-			l.Close()
-		})
-	}
 
-	level.Info(logger).Log("msg", "starting compact node")
+	level.Info(logger).Log("msg", "starting downsample node")
```

Comment:
I'm not sure, shouldn't you call `metricHTTPListenGroup` here?

Reply:
Thanks for sharp eye here! But no metricHTTPListenGroup is intentional here. This is only CLI command.